src/utilities/ScrollToTop.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+
+const ScrollToTop = () => {
```

Comment:
The component's name is `ScrollToTop` but the behavior differs from the name. Can you rename it properly?